src/Components/EditProject/EditProject.tsx

```diff
@@ -1,30 +1,39 @@
 import axios from 'axios';
-import React, { useContext, useEffect, useState } from 'react'
-import { useForm } from 'react-hook-form'
-import { useParams } from 'react-router-dom'
+import { useContext, useEffect } from 'react';
+import { useForm } from 'react-hook-form';
+import { useNavigate, useParams } from 'react-router-dom';
 import { AuthContext } from '../../Context/AuthContext';
 
 export default function EditProject() {
-  const {id}:any=useParams
+  const {id}:any=useParams();
 
    const {register,handleSubmit,formState:{errors},setValue}=useForm();
    const{baseUrl,reqHeaders}:any=useContext(AuthContext)
-   // uncompleted function
+ const navigate=useNavigate()
+ 
    const updatePro = (data)=>{
     axios.put (`${baseUrl}/project/${id}`,data,{headers:reqHeaders}).then((response)=>
```

Comment:
you can provide toast message upon updating the project details, as it provides a good user experience of what is happening

Reply:
You also need to comment or remove the logs before committing them to the branch